server/src/user/user.controller.ts

```diff
@@ -12,4 +18,13 @@ export class UserController {
   getMe(@GetUser('id') userId: number) {
     return this.userService.getMe(userId);
   }
+
+  @UseGuards(AuthGuard)
+  @Get(':houseId/todos')
+  getTodos(
+    @GetUser('id') userId: number,
+    @Param('houseId', ParseIntPipe) houseId: number,
+  ) {
+    return this.userService.getTodos(userId, houseId);
+  }
```

Comment:
I feel it more makes sense to place under `house.controller.ts` because me is more focusing of user data, not a house. 
So maybe like `houses/[id]/todos` is better. What do you think?

Reply:
I agree!
I fixed it!
https://github.com/HAK2024/fairy-share/pull/61/commits/655a7d2b00bf3c632885db38c14f990efd73ec2b